sklearn/utils/validation.py

```diff
@@ -547,14 +548,38 @@ def check_array(array, accept_sparse=False, dtype="numeric", order=None,
         msg = ("Data with input dtype %s was converted to %s%s."
                % (dtype_orig, array.dtype, context))
         warnings.warn(msg, DataConversionWarning)
+
+    # Indices Datatype regulation
+    _check_large_sparse(array, accept_large_sparse)
+
     return array
 
 
+def _check_large_sparse(array, accept_large_sparse=False):
+    """
```

Comment:
This first line should contain the summary of the function description, and it should begin with a verb in present tense.

Reply:
ohhk,

i will modify accordingly